delocate/tools.py

```diff
@@ -844,12 +845,27 @@ def zip2dir(
             os.utime(extracted_path, (modified_time, modified_time))
 
 
+ZIP_TIMESTAMP_MIN = 315532800  # 1980-01-01 00:00:00 UTC
+DateTuple = Tuple[int, int, int, int, int, int]
+
+
+def get_zip_datetime(
```

Comment:
I'd like this to be a private function (add a single leading underscore.)  You should also add a short docstring to clarify when it returns what value, and maybe link to the standard.